module/account/src/Application/Controller/Api/Role/RoleChangeAction.php

```diff
@@ -113,7 +113,6 @@ public function __invoke(Role $role, Request $request): Response
 
                 $command = new UpdateRoleCommand(
```

Comment:
It seems like the behavior is changed but the HTTP schema remains the same(`@SWG\Schema(ref="#/definitions/role")`).
It's a bit confusing I guess since the API is non-deterministic as it's not validating the entire request but just the 'picked' pieces